tck/src/main/java/org/eclipse/microprofile/config/tck/ConfigProviderTest.java

```diff
@@ -88,28 +88,18 @@ public static void setupCheck(){
 
     @Test
     public void testEnvironmentConfigSource() {
-        Map<String, String> env = System.getenv();
-        Properties properties = System.getProperties();
-        for (Map.Entry<String, String> envEntry : env.entrySet()) {
-            String key = envEntry.getKey();
-            if(!properties.containsKey(key)) {
-                String value = envEntry.getValue();
-                String cfgValue = config.getValue(key, String.class);
-                Assert.assertEquals(value, cfgValue, "The property "+key+" did not have the expected value.");
-            }
-        }
+        String value = System.getenv().get("MP_TCK_ENV_DUMMY");
+        Assert.assertNotNull(value);
+        Assert.assertEquals("dummy", value);
+        Assert.assertEquals(value, config.getValue("mp.tck.env.dummy", String.class));
     }
 
     @Test
     public void testPropertyConfigSource() {
-        Properties properties = System.getProperties();
-
-        for (Map.Entry<Object, Object> propEntry : properties.entrySet()) {
-            String propValue = (String) propEntry.getValue();
-            if (propValue != null && propValue.length() > 0) {
-                Assert.assertEquals(propValue, config.getValue((String) propEntry.getKey(), String.class));
-            }
-        }
+        String value = System.getProperties().getProperty("mp.tck.prop.dummy");
+        Assert.assertNotNull(value);
+        Assert.assertEquals("dummy", value);
+        Assert.assertEquals(value, config.getValue("mp.tck.prop.dummy", String.class));
```

Comment:
Same as above. Can you also check some built in jvm properties such as java.home, user.name?